mkdocs/docs/getting-started/identity/identity-types.md

```diff
@@ -0,0 +1,128 @@
+# Identity Types
+
+In iden3 protocol there are two types of identities, which differ in many aspects:
+
+1. **Regular Identity**, which is generated from three identity trees and controlled by Baby JubJub keys.
+2. **Ethereum-controlled Identity**, which is primarily controlled by Ethereum account from which it's Genesis State and
+   Identifier are derived.
+
+## Regular Identity
+
+Regular identity is created from three merkle trees (Genesis State is a hash of Identity SMT Roots). This identity is
+primarily controlled by Baby JubJub keys. At least one BJJ public key must be added into Claims Tree during the identity
+creation.
+
+```
+genesisState = Hash(ClaimsTreeRoot || RevocationsTreeRoot || RootsTreeRoot)
+
+genesisId = idType + genesisStateCut + checksum
+```
+
+where:
+
+* idType - identifier of DID method and blockchain network & subnetwork, 2 bytes
+* genesisStateCut - first 27 bytes of genesisState, 27 bytes
+* checksum - control checksum, 2 bytes
+
+### Limitations of Regular Identity
+
+Currently, adding Ethereum key to the Claims Tree and using it for authentication and proving is not practical, because
+it's very computationally expensive to verify ECDSA signatures in zk-circuits. Also, having user's Ethereum address
+there's no way to get user identifier from it, so dApps would need to authenticate user additionally to get the
+identifier.
```

Comment:
```suggestion
Currently, adding an Ethereum key to the Claims Tree and using it for authentication and proving is not practical, because
it's very computationally expensive to verify ECDSA signatures in zk-circuits. Also, having the user's Ethereum address
there's no way to get a user identifier from it, so dApps would need to authenticate the user additionally to get the
identifier.
```